kernel/src/main/java/org/kframework/parser/ParserUtils.java

```diff
@@ -283,7 +308,7 @@ private String loadDefinitionText(File definitionFile) {
         Set<File> requiredFiles = new HashSet<>();
         Context context = new Context();
         if (autoImportDomains)
-            previousModules.addAll(loadModules(new HashSet<>(), context, Kompile.REQUIRE_PRELUDE_K, source, currentDirectory, lookupDirectories, requiredFiles, kore, preprocess, leftAssoc));
+            previousModules.addAll(loadModules(new HashSet<>(), context, Kompile.REQUIRE_PRELUDE_K, Source.apply("auto imported prelude"), currentDirectory, lookupDirectories, requiredFiles, kore, preprocess, leftAssoc));
```

Comment:
Not sure what's going on here.

Reply:
Was this a bug before?

Reply:
When using -v, it would show that you are including the main file twice.
So yeah... a visual bug.